app/src/docs/SliderRating.doc.tsx

```diff
@@ -1,10 +1,14 @@
 import * as React from 'react';
 import {
-    BaseDocsBlock, DocExample, EditableDocContent, UUI3,
+    BaseDocsBlock, DocExample, EditableDocContent, TDocsGenType, UUI3,
 } from '../common';
 
 export class SliderRatingDoc extends BaseDocsBlock {
     title = 'SliderRating';
+
+    // TODO: no such component in "@epam/uui"
+    override getDocsGenType = (): TDocsGenType => ('@epam/loveship:SliderRatingProps');
```

Comment:
get types from loveship package here. Because it's a skin specific component which will not be moved to the uui package.

Reply:
done